modules/audio.nix

```diff
@@ -0,0 +1,28 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}: {
+  options.x1e.audio.enable =
+    lib.mkEnableOption
+    ''      Enable audio on the Lenovo Yoga Slim 7x.
+              HIGHLY EXPERIMENTAL, and if improperly used *WILL* damage the speakers.
+              Keep audio low (5-10%) and also expect some distortion.'';
+
+  config =
+    {x1e.audio.enable = lib.mkDefault false;}
```

Comment:
There is no need for this AFAIK, `mkEnableOption` already defaults to false. See: https://nixos.org/manual/nixos/stable/#sec-option-declarations-util-mkEnableOption

Reply:
I'd rather be explicit about what I want for a default, even if it's implied.